### PR TITLE
[FEATURE] Support preserving !important

### PR DESCRIPTION
**Pull request by penn5**

### CODE REVIEW

**Review by JakeQZ — CHANGES_REQUESTED** [review]
Hi @penn5,

Thanks for contributing.  I think I can think of a situation where this would be useful: you retain the original CSS to cover the edge cases where Emogrifier does not quite (or cannot) work, but still want the inlined CSS to take precedence otherwise (and have some `!important` declarations in the original CSS).

The code changes look fine, but could you
- Add some unit tests to verify the behaviour is as expected;
- Update the README to document the new feature;
- Add an entry to the CHANGELOG?

Thanks <3

File: src/CssInliner.php [hunk 1]
```diff
@@ -315,6 +324,20 @@ public function setDebug(bool $debug): self
         return $this;
     }
 
+    /**
+     * Configure whether to remove the !important annotation from inline styles or not.
+     * Must be used before inlineCss is called.
+     *
+     * @param bool $removeImportantAnnotationFromAllInlineStyles set to true to remove the !important annotation
+     * @return self
```
Comment (JakeQZ) [inline src/CssInliner.php:332]: I think the `phpdoc_separation` seperation error is coming from here, and needs a empty PHPDoc line between `@param` and `@return`.

File: src/CssInliner.php [hunk 2]
```diff
@@ -315,6 +324,20 @@ public function setDebug(bool $debug): self
...
+     */
+    public function setRemoveImportantAnnotationFromAllInlineStyles(bool $removeImportantAnnotationFromAllInlineStyles): self
```
Comment (JakeQZ) [inline src/CssInliner.php:334]: There's a line limit length of 120 characters.

**Review by JakeQZ — CHANGES_REQUESTED** [review]
PS. I've enabled the running of the CI tests, which have picked up a couple of formatting issues.

**Comment by penn5** [comment]
I don't think I have time to learn whatever test framework you use, sorry.

**Comment by oliverklee** [comment]
@penn5 If you want help/support for running and writing the tests, please let us know. We're willing to help.

**Comment by penn5** [comment]
Unfortunately I don't know anything about php unit testing and I don't really have time to learn. Sorry.

**Comment by oliverklee** [comment]
Closing then. If at any time you would like to give a tests a go, please feel free to reopen this PR. We'll be happy to support you with the tests.

**Pull request closed without merging**